VERSION

```diff
@@ -1 +1 @@
-1.4.0
\ No newline at end of file
+1.5.0
```

Comment:
@ChuckKollar @AlanSimmons I was not requested as a reviewer. However, I should be tagged as one for tracking purposes even if sometimes I may not familiar to the code changes. 

WRT this PR, we should use the version 1.4.1 instead of 1.5.0. We use the [Semantic Versioning 2.0.0](https://semver.org/) as a general guideline. You don't have to bump the version number, consider it optional. I would take care of that by bumping the number in `main` branch then sync into `dev-integrate` for release purposes.